hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -143,7 +143,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     try {
       // check Acl
       if (ozoneManager.getAclsEnabled()) {
-        checkAcls(ozoneManager, OzoneObj.ResourceType.VOLUME,
+        checkAcls(ozoneManager, OzoneObj.ResourceType.BUCKET,
```

Comment:
This will require some additional changes on OzoneNativeAuthorizer as in the case of CREATE, ozone bucketmanager does not have any acl info wrt. the bucket that has not been created yet.

Reply:
Created https://issues.apache.org/jira/browse/HDDS-2191 to take care of this